mozilla/foundation.mozilla.org
# Cleanup/7392 remove softwareproductpage

To test:

1. Setup project with current production setup: `git switch main`, then `inv copy-stage-db`.
2. Run new migration on the product setup: `git switch cleanup/7392-remove-softwareproductpage`, then `inv migrate`. This should pass without issue. 
3. Start the containers `docker compose up` and visit http://localhost:8000/en/privacynotincluded/. Click around and make sure PNI is still working as expected. Try also visiting it in Wagtail. 
4. Check that a new database also works. <kbd>Control</kbd> + <kbd>C</kbd> then `inv new-db`. This should work without issue.
5. Start the containers again `docker compose up`. Again, visit http://localhost:8000/en/privacynotincluded/ and click around to make sure everything still works as expected. 


***

Closes #7392 
Related PRs/issues #

**Link to sample test page**: https://foundation-s-cleanup-73-alh1lr.herokuapp.com/en/privacynotincluded/
No sample pages. The page type is removed. But, PNI should still be working. 


## Checklist

_Remove unnecessary checks_

**Tests**
- [x] Is the code I'm adding covered by tests? -> No, the tests are removed

**Changes in Models:**
- [x] Did I update or add new fake data? -> No, the fake data is removed
- [x] Did I squash my migration? No, only a single migration file added
- [x] [Are my changes backward-compatible](https://github.com/mozilla/foundation.mozilla.org/blob/main/docs/workflow.md#django-migrations-what-to-do-when-working-on-backward-incompatible-migrations). If not, did I schedule a deploy with the rest of the team? -> Page usage has already been removed in the past. So this should be backward compatible. 

**Documentation:**
- [x] Is my code documented? -> No
- [x] Did I update the READMEs or wagtail documentation? -> No
